frontend/src/store/uam/actions.ts

```diff
@@ -24,11 +24,11 @@ const getUsers = createAsyncThunk<
 
 const getGroups = createAsyncThunk<
   EntityPagination<GroupsItemResponseDto>,
-  void,
+  EntityPaginationRequestQueryDto,
   AsyncThunkConfig
->(ActionType.GET_GROUPS, async (_, { extra }) => {
+>(ActionType.GET_GROUPS, async ({ page, count }, { extra }) => {
   const { groupsApi } = extra;
-  const groups = await groupsApi.getAll();
+  const groups = await groupsApi.getPaginated({ page, count });
```

Comment:
```suggestion
  const groups = await groupsApi.getAll({ page, count });
```
it should be a `getAll
pagination is optional
fix it for all (if we have it somewhere else)

Reply:
@what1s1ove But in Back-End of group pagination, there's no `getAll` method, only `getPaginated`. Should I add the back-end for it?

Reply:
right. should be only `getAll`